src/ToggleButton/Option.js

```diff
@@ -0,0 +1,253 @@
+// @flow
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+import Menu from '../Menu';
+import ArrowDropdown from '../svg-icons/arrow-drop-down';
+import withStyles from '../styles/withStyles';
+import createStyleSheet from '../styles/createStyleSheet';
+import { fade } from '../styles/colorManipulator';
+import common from '../colors/common';
+import grey from '../colors/grey';
+
+/**
+ * Represents an option within a Toggle Button
+ *
+ */
+
+export const styleSheet = createStyleSheet('MuiOption', {
+  root: {
+    backgroundColor: 'transparent',
+    borderRadius: 0,
+    fontWeight: 600,
+    fontSize: 14,
+    float: 'left',
```

Comment:
That's the first usage of the floating property. Can we stick to the flexbox model? That's more performant and more flexible.